docs/guides/postgres/backup/kubestash/customization/index.md

```diff
@@ -0,0 +1,414 @@
+        namespace: demo
+  sessions:
+    - name: frequent-backup
+      scheduler:
+        schedule: "*/5 * * * *"
+        jobTemplate:
+          backoffLimit: 1
+      repositories:
+        - name: gcs-postgres-repo
+          backend: gcs-backend
+          directory: /postgres
+          encryptionSecret:
+            name: encrypt-secret
+            namespace: demo
+      addon:
+        name: postgres-addon
+        tasks:
+          - name: logical-backup
+            params:
+              args: --clean
+```
+
+
+### Passing a target database to the backup process
+
+KubeStash PostgreSQL addon uses the [pg_dumpall](https://www.postgresql.org/docs/current/app-pg-dumpall.html) command by default for backups. For a single database backup, you need to rewrite the dump command. You can do this by setting `backupCmd` to [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) under the `addon.tasks[*].params` section and specifying the database name using the `args` parameter in the same section.
+
+The below example shows how you can set `pg_dump` and pass target database name during backup. 
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: BackupConfiguration
+metadata:
+  name: sample-postgres-backup
+  namespace: demo
+spec:
+  target:
+    apiGroup: kubedb.com
+    kind: Postgres
+    namespace: demo
+    name: sample-postgres
+  backends:
+    - name: gcs-backend
+      storageRef:
+        namespace: demo
+        name: gcs-storage
+      retentionPolicy:
+        name: demo-retention
+        namespace: demo
+  sessions:
+    - name: frequent-backup
+      scheduler:
+        schedule: "*/5 * * * *"
+        jobTemplate:
+          backoffLimit: 1
+      repositories:
+        - name: gcs-postgres-repo
+          backend: gcs-backend
+          directory: /postgres
+          encryptionSecret:
+            name: encrypt-secret
+            namespace: demo
+      addon:
+        name: postgres-addon
+        tasks:
+          - name: logical-backup
+            params:
+              backupCmd: pg_dump
+              args: testdb
+```
+
+> **WARNING**: Make sure that your provided database has been created before taking backup.
+
+### Using multiple repositories
+
+You can configure multiple repositories for the same backend. For example, if you want to back up the `/postgres` directory using the `gcs-postgres-repo` repository, you can also back up another directory, such as `/postgres-copy`, by using a different repository, like `gcs-postgres-repo-2`.
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: BackupConfiguration
+metadata:
+  name: sample-postgres-backup
+  namespace: demo
+spec:
+  target:
+    apiGroup: kubedb.com
+    kind: Postgres
+    namespace: demo
+    name: sample-postgres
+  backends:
+    - name: gcs-backend
+      storageRef:
+        namespace: demo
+        name: gcs-storage
+      retentionPolicy:
+        name: demo-retention
+        namespace: demo
```

Comment:
You can  provide an example of using different backends instead of the same backend.